stdlib/public/SwiftShims/RuntimeShims.h

```diff
@@ -59,6 +59,13 @@ int _swift_stdlib_putc_stderr(int C);
 SWIFT_RUNTIME_STDLIB_API
 __swift_size_t _swift_stdlib_getHardwareConcurrency(void);
 
+SWIFT_RUNTIME_STDLIB_API
```

Comment:
Should this be API? Does INTERNAL fit the bill better?

Reply:
mmm, is this meant to be used by users of Swift?

Reply:
If we don't intend on `demangle` being `@inlinable` then INTERNAL should be fine, but I don't whether we want it to be `@inlinable` or not.

Reply:
I don't think it needs to be inlinable. We wouldn't want this to be an exported entry point, so STDLIB_SPI is probably more appropriate.

Reply:
STDLIB_SPI is still exported no?